### DISCUSSION TITLE
Add an accuracy benchmark next to the performance benchmark

### DISCUSSION THREAD

**Discussion by abique** [kfrlib/kfr#215] [Ideas]
Hi,

The performance benchmarks shows great results, but on the other hand let us wonder if the accuracy is as good.

Thank you,
Alex.

**Comment by dancasarin** [comment]
Hi,

KFR uses highly optimized radix-4 DIF FFT, so the accuracy is almost the same as in all similar implementations.
I agree, it would be right to test the accuracy as well as the performance.
As of now, all transcendental functions in KFR are already tested using MPFR and have proven accuracy.
